tools/generateNetCoreSamplesInclude.linq

```diff
@@ -80,7 +80,7 @@ private void CreateYamlDeserializer()
     yamlDeserializer = builder.Build();
 }
 
-static Deserializer yamlDeserializer;
+static IDeserializer yamlDeserializer;
```

Comment:
Why was this change necessary?

Reply:
Because of a change YamlDotNet introduced. I've run into this too.